bazelrio/scripts/wpiformat/wpiformat.py

```diff
@@ -0,0 +1,5 @@
+from os import system
+from sys import exit
+
+if __name__ == "__main__":
+    exit(system("wpiformat"))
```

Comment:
This doesn't feel right. I bet this completely ignores your what you pip installed, and uses the non-hermetic version.

```
import wpiformat
import os

if __name__ == "__main__":
    root = os.environ["BUILD_WORKSPACE_DIRECTORY"]
    os.chdir(root)
    wpiformat.main()
```